tests/ui/stable_sort_primitive.fixed

```diff
@@ -0,0 +1,18 @@
+// run-rustfix
+#![warn(clippy::stable_sort_primitive)]
+
+fn main() {
+    // positive examples
+    let mut vec = vec![1, 3, 2];
+    vec.sort_unstable();
+    let mut vec = vec![false, false, true];
```

Comment:
Should we add a test for chars as well?

Reply:
Also test cases for types that deref to a slice of primitives.

Reply:
I believe `Vec` derefs to a slice, so I think the tests already have that last bit covered, unless I don't understand what you're getting at.

Reply:
No, you are correct. My bad.

Reply:
@JarredAllen Hey should we add test cases for list of tuples such as: `vec![(2, 1), (1, 3), (3, 2)]`

Reply:
@krishna-veerareddy I added test cases for a list of tuples and also for one of arrays.